showcase/src/Tables/Book.elm

```diff
@@ -45,7 +45,7 @@ someFilters =
 
 
 books =
-    [ { author = "Dan Brown"
+    [ { author = "Dan Lorem ipsum dolor sit amet consectetur adipisicing elit. "
```

Comment:
Don't break my heart like this, there is the Narnia book, just change the default filter to always show it! :sweat_smile:

Reply:
sorry this wasn't meant to be pushed lol